go/vt/wrangler/materializer_test.go

```diff
@@ -254,15 +268,26 @@ func TestMigrateVSchema(t *testing.T) {
 	require.NoError(t, err)
 	vschema, err := env.wr.ts.GetSrvVSchema(ctx, env.cell)
 	require.NoError(t, err)
-	got := fmt.Sprintf("%v", vschema)
-	want := []string{`keyspaces:{key:"sourceks" value:{}}`,
-		`keyspaces:{key:"sourceks" value:{}} keyspaces:{key:"targetks" value:{tables:{key:"t1" value:{}}}}`,
-		`rules:{from_table:"t1" to_tables:"sourceks.t1"}`,
-		`rules:{from_table:"targetks.t1" to_tables:"sourceks.t1"}`,
-	}
-	for _, wantstr := range want {
-		require.Contains(t, got, wantstr)
+
+	require.Equal(t, vschema.Keyspaces["sourceks"], &vschemapb.Keyspace{})
```

Comment:
This looks very similar (if not identical) to the change in the previous test. Maybe we can extract it into a utility function?